ntoskrnl/include/internal/ex.h

```diff
@@ -79,9 +79,9 @@ VOID NTAPI ExpDebuggerWorker(IN PVOID Context);
 #define HANDLE_LOW_BITS     (PAGE_SHIFT - 3)
 #define HANDLE_HIGH_BITS    (PAGE_SHIFT - 2)
 #endif
-#define HANDLE_TAG_BITS     (2)
-#define HANDLE_INDEX_BITS   (HANDLE_LOW_BITS + 2*HANDLE_HIGH_BITS)
-#define KERNEL_FLAG_BITS    (sizeof(PVOID)*8 - HANDLE_INDEX_BITS - HANDLE_TAG_BITS)
+#define HANDLE_TAG_BITS     2
+#define HANDLE_INDEX_BITS   (HANDLE_LOW_BITS + 2 * HANDLE_HIGH_BITS)
+#define KERNEL_FLAG_BITS    (sizeof(ULONG_PTR) * 8 - HANDLE_INDEX_BITS - HANDLE_TAG_BITS)
```

Comment:
Just to be sure, why the "PVOID" --> "ULONG_PTR" change? Is it because the variable that stores these bits has the ULONG_PTR type?

Reply:
Yes, to match.